msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -89,6 +89,9 @@ private void initializeSharedPreferenceFileManager(@NonNull final Context contex
                 new StorageHelper(context));
     }
 
+    /**
```

Comment:
Javadoc comments in this file already exists in its interface file(s). If we need to update, shouldn't we update it there instead?